io/disk/port_bounce.py

```diff
@@ -191,23 +199,49 @@ def verify_port_enable(self, test_ports):
                 msg = "Port %s is failed to enable" % port
                 self.failure_list[port] = msg
 
-    def verify_port_toggle_host(self, status):
```

Comment:
Any reason this is removed ?

Reply:
this was mistake while editing the file will add it again. :-)